tasks/gulp/copy-to-destination.js

```diff
@@ -25,5 +30,24 @@ gulp.task('copy-files', () => {
       autoprefixer
     ], {syntax: require('postcss-scss')}))
     .pipe(scssFiles.restore)
+    .pipe(yamlFiles)
+    .pipe(map(function (file, done) {
+      let componentName = path.dirname(file.path).split(path.sep).slice(-1).toString()
+      let componentPath = path.join(configPaths.components, componentName, `${componentName}.yaml`)
+
+      try {
+        let json = yaml.safeLoad(fs.readFileSync(componentPath), {encoding: 'utf8', json: true})
+        let paramsJson = json.params // We only want the 'params' data from component yaml
+        file.contents = Buffer.from(JSON.stringify(paramsJson, null, 4))
+      } catch (e) {
+        console.log('ENOENT: no such file or directory: ', file)
```

Comment:
This should probably be `console.error`. Also the `try` section has 3 lines in it, so we don't know for sure that this is the error. I'd recommend trying to reduce the `try` to just opening the file.

Reply:
I've split up the error messages more, have a look.